README.md

```diff
@@ -108,16 +119,23 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
-
+```
+When a function is called (or invoked), the code inside will execute. This can happen when an event occurs or when it is called within a block of JS code.
+$("p").on("click", doSomething);
 ```
```

Comment:
You described invoking a function properly, but the example you provided is actually referencing a function (i.e., no parentheses).